src/main/resources/fester.yaml

```diff
@@ -130,6 +130,58 @@ paths:
               schema:
                 type: string
                 format: binary
+  /thumbnails:
+    post:
+      tags: [Collection]
+      summary: Post a Collection to add thumbnails
+      description: "This endpoint accepts CSV data to use in the addition of a 
+        thumbnail to a collection"
```

Comment:
It would be more accurate to say that this is adding thumbnails to each work in a collection

Reply:
Modified wording